packages/flutter/lib/src/gestures/debug.dart

```diff
@@ -10,11 +10,20 @@ import 'package:flutter/foundation.dart';
 /// Whether to print the results of each hit test to the console.
 ///
 /// When this is set, in debug mode, any time a hit test is triggered by the
-/// [GestureBinding] the results are dumped to the console.
+/// [GestureBinding], the results are dumped to the console.
 ///
 /// This has no effect in release builds.
 bool debugPrintHitTestResults = false;
 
+/// Whether to print the details of each mouse hover event to the console.
+///
+/// When this is set, in debug mode, any time a mouse hover event is triggered
+/// by the [GestureBinding], the results are dumped to the console.
+///
+/// This has no effect in release builds, and only applies to mouse hover events
+/// (touch events can't hover).
```

Comment:
in theory we will eventually support that (some hardware can detect touches before they hit the screen, and they'd be considered hover events).